src/tests/end2end/D3D12ResidencyManagerTests.cpp

```diff
@@ -172,7 +172,7 @@ TEST_F(D3D12ResidencyManagerTests, CreateResidencySet) {
         ResidencySet setA;
         ASSERT_SUCCEEDED(setA.Insert(allocation->GetMemory()));
         ResidencySet setB(setA);
-        EXPECT_EQ(setA.Insert(allocation->GetMemory()), S_FALSE);
+        EXPECT_EQ(setA.Insert(allocation->GetMemory()), S_OK);
```

Comment:
What do you think about changing `ResidencySet::Insert` to ResidencyList::Add`? This would follow your idea much better IMHO.

Reply:
sgtm.